academic-events/requirements.txt

```diff
@@ -1,44 +1,7 @@
-attrs==20.3.0
```

Comment:
Nice tidy up of these requirements!

Reply:
Good job on the PHQ version update- Thanks!